lib/states/state.js

```diff
@@ -192,6 +192,11 @@ var State = Eventable.extend(function(self, name, opts) {
     :param boolean opts.continue_session:
         whether or not this is the last state in a session. Default is ``true``.
         May also be a function, which may return its result via a promise.
+    :param object opts.helper_metadata:
+        additional helper metadata to set on the reply sent to the user.
+        Primarily useful for setting voice metadata for messages destined to
+        be sent as voice calls. Default is `null`. May also be a function,
+        which may return its result via a promise.
```

Comment:
We seem to be only documenting the primary type in this docstrings (send_reply, continue_session work the same way).